eZ/Publish/Core/IO/ConfigScopeChangeAwareIOService.php

```diff
@@ -33,6 +33,9 @@ public function __construct(
         $this->configResolver = $configResolver;
         $this->innerIOService = $innerIOService;
         $this->prefixParameterName = $prefixParameterName;
+
+        // set initial prefix on inner IOService
+        $this->setPrefix($this->configResolver->getParameter($this->prefixParameterName));
```

Comment:
If you have to do it here, the service should be `lazy` to avoid getting parameter from uninitialized Config Resolver.